pages/index.vue

```diff
@@ -12,6 +11,15 @@ const articles = await findArticlePages('').then(({ data }) =>
   normalizeFindArticlePagesResponse(data)
 )
 
+const newsletterSubmitEvent = (e) => {
+  //emitted newsletter submit event, @Matt, not exactly sure how to get this work like you mentioned.
```

Comment:
@walsh9 Was having trouble with the gtag event here... but I emitted it up to the page like you asked.